### PR TITLE
Rerun cytotable merge

### PR DESCRIPTION
**Pull request by MikeLippincott**
This PR reruns the cytotable and pycytominer processing for PBMCs.

### CODE REVIEW

File: Plate_2_data/4.processing_features/scripts/1.annotate_sc_PBMC.py
Comment (jenna-tomkinson) [inline Plate_2_data/4.processing_features/scripts/1.annotate_sc_PBMC.py:40]: Recommend removing one of these code comments as they are duplicates.

File: Plate_2_data/4.processing_features/scripts/1.annotate_sc_plate2.py
Comment (jenna-tomkinson) [inline Plate_2_data/4.processing_features/scripts/1.annotate_sc_plate2.py:1]: Same as comment for previous notebook, recommend removing this one if not needed since you have notebooks for each cell type specifically.

File: Plate_2_data/4.processing_features/scripts/1.annotate_sc_plate2_SHSY5Y.py
Comment (jenna-tomkinson) [inline Plate_2_data/4.processing_features/scripts/1.annotate_sc_plate2_SHSY5Y.py:8]: Same comment here regarding running cells.

File: Plate_2_data/4.processing_features/scripts/2.combine_sc_runs_plate2.py
Comment (jenna-tomkinson) [inline Plate_2_data/4.processing_features/scripts/2.combine_sc_runs_plate2.py:1]: Same comment here.

File: Plate_2_data/4.processing_features/scripts/2.combine_sc_runs_plate2_PBMC.py
Comment (jenna-tomkinson) [inline Plate_2_data/4.processing_features/scripts/2.combine_sc_runs_plate2_PBMC.py:34]: Looks like this cell wasn't ran, recommend rerunning to have all cells in order.

File: Plate_2_data/4.processing_features/scripts/2.combine_sc_runs_plate2_PBMC.py
Comment (jenna-tomkinson) [inline Plate_2_data/4.processing_features/scripts/2.combine_sc_runs_plate2_PBMC.py:13]: Recommend adding a short code comment on what this package does. It looks like it is very useful for memory management given how big each batch is to combine.

File: Plate_2_data/4.processing_features/scripts/2.combine_sc_runs_plate2_SHSY5Y.py
Comment (jenna-tomkinson) [inline Plate_2_data/4.processing_features/scripts/2.combine_sc_runs_plate2_SHSY5Y.py:8]: Same comment here.

File: Plate_2_data/4.processing_features/scripts/2.combine_sc_runs_plate2_SHSY5Y.py
Comment (jenna-tomkinson) [inline Plate_2_data/4.processing_features/scripts/2.combine_sc_runs_plate2_SHSY5Y.py:13]: Recommend removing import if not used in the file.

File: Plate_2_data/4.processing_features/scripts/3.normalize_sc_plate2.py
Comment (jenna-tomkinson) [inline Plate_2_data/4.processing_features/scripts/3.normalize_sc_plate2.py:1]: Same comment here.

File: Plate_2_data/4.processing_features/scripts/3.normalize_sc_plate2_SHSY5Y.py
Comment (jenna-tomkinson) [inline Plate_2_data/4.processing_features/scripts/3.normalize_sc_plate2_SHSY5Y.py:8]: Same comment here.

File: Plate_2_data/4.processing_features/scripts/4.feature_select_sc_plate2.py
Comment (jenna-tomkinson) [inline Plate_2_data/4.processing_features/scripts/4.feature_select_sc_plate2.py:1]: Same comment here.

File: Plate_2_data/4.processing_features/scripts/4.feature_select_sc_plate2_PBMC.py
Comment (jenna-tomkinson) [inline Plate_2_data/4.processing_features/scripts/4.feature_select_sc_plate2_PBMC.py:8]: Recommend making sure all code cells are ran prior to nbconverting.

File: Plate_2_data/4.processing_features/scripts/4.feature_select_sc_plate2_PBMC.py
Comment (jenna-tomkinson) [inline Plate_2_data/4.processing_features/scripts/4.feature_select_sc_plate2_PBMC.py:58]: Recommend defining at the start of the notebook.

File: Plate_2_data/4.processing_features/scripts/4.feature_select_sc_plate2_PBMC.py
Comment (jenna-tomkinson) [inline Plate_2_data/4.processing_features/scripts/4.feature_select_sc_plate2_PBMC.py:139]: Recommend being more explicit on what is happening in this last code cell. Why do we need to see the length of the metadata columns and feature columns?

File: Plate_2_data/4.processing_features/scripts/4.feature_select_sc_plate2_SHSY5Y.py
Comment (jenna-tomkinson) [inline Plate_2_data/4.processing_features/scripts/4.feature_select_sc_plate2_SHSY5Y.py:8]: Same comment here.

File: Plate_2_data/4.processing_features/processing_features_plate2_SHSY5Y.sh
Comment (jenna-tomkinson) [inline Plate_2_data/4.processing_features/processing_features_plate2_SHSY5Y.sh:37]: I believe that this should have a 5 as the prefix, but please correct this.
```suggestion
time python scripts/5.feature_select_sc_plate2_SHSY5Y.py
```

File: Plate_2_data/4.processing_features/processing_features_plate2_PBMC.sh
Comment (jenna-tomkinson) [inline Plate_2_data/4.processing_features/processing_features_plate2_PBMC.sh:22]: Recommend updating code comment above this line to not include extract image features if the python file is not included in the file.

**Review by jenna-tomkinson — APPROVED** [review]
Nice PR! I have included some small comments, LGTM after addressing.

File: Plate_2_data/4.processing_features/processing_features_plate2_PBMC.sh
Comment (MikeLippincott) [reply Plate_2_data/4.processing_features/processing_features_plate2_PBMC.sh:11]: module purge removes loaded modules to ensure that the script runs in a clean cluster env and only uses modules loaded by the script. This is a modules specific command

File: Plate_2_data/4.processing_features/scripts/0.merge_sc_plate2.py
Comment (MikeLippincott) [reply Plate_2_data/4.processing_features/scripts/0.merge_sc_plate2.py:1]: Removed

File: Plate_2_data/4.processing_features/scripts/4.feature_select_sc_plate2_PBMC.py
Comment (MikeLippincott) [reply Plate_2_data/4.processing_features/scripts/4.feature_select_sc_plate2_PBMC.py:139]: Just to understand the number of features remaining!

**Pull request merged**
